editor/animation_track_editor.cpp

```diff
@@ -2942,8 +2942,7 @@ void AnimationTrackEdit::gui_input(const Ref<InputEvent> &p_event) {
 		path_popup->set_position(get_screen_position() + path_rect.position - theme_ofs);
 		path_popup->set_size(path_rect.size);
 		path_popup->popup();
-		path->grab_focus();
-		path->set_caret_column(path->get_text().length());
+		path->grab_focus_edit();
```

Comment:
I forgot to add this line, and I don't want to make more noise! 😄
`path->set_caret_column(path->get_text().length());`